src/components/Palette/utils.ts

```diff
@@ -0,0 +1,23 @@
+import type {PaletteOption} from './Palette';
+
+export function getPaletteRows(options: PaletteOption[], columns: number): PaletteOption[][] {
```

Comment:
Let's add some basic tests for this function

Reply:
Added tests